ts/Extensions/PriceIndication.ts

```diff
@@ -144,9 +146,14 @@ addEvent(Series, 'afterRender', function (): void {
             crop;
 
         if (lastPrice && lastPrice.enabled) {
-
             yAxis.crosshair = yAxis.options.crosshair = seriesOptions.lastPrice;
 
+            if (!series.chart.styledMode) {
+                yAxis.crosshair = yAxis.options.crosshair = merge({
+                    color: series.color // default color from the series #14888
+                }, seriesOptions.lastPrice);
```

Comment:
Can we simplify this a little? In general `merge`s are time-consuming, so perhaps this could be easier and faster? Something like this:
```js
yAxis.crosshair.color = yAxis.options.crosshair.color = seriesOptions.lastPrice.color || series.color
```